src/domains/wallet-app/04_apim_wallet.tf

```diff
@@ -19,3 +19,52 @@ module "apim_wallet_product" {
 
   policy_xml = file("./api_product/_base_policy.xml")
 }
+
+#################################################
+## API wallet service                          ##
+#################################################
+locals {
+  apim_wallet_service_api = {
+    display_name          = "pagoPA - wallet API"
+    description           = "API to support wallet service"
+    path                  = "wallet-service"
+    subscription_required = true
+    service_url           = null
+  }
+}
+
+# Wallet service APIs
+resource "azurerm_api_management_api_version_set" "wallet_service_api" {
+  name                = format("%s-wallet-service-api", local.project)
+  resource_group_name = local.pagopa_apim_rg
+  api_management_name = local.pagopa_apim_name
+  display_name        = local.apim_wallet_service_api.display_name
+  versioning_scheme   = "Segment"
+}
+
+module "apim_wallet_service_api_v1" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//api_management_api?ref=v6.3.0"
+
+  name                  = format("%s-wallet-service-api", local.project)
```

Comment:
mmm local.project has `wallet`, so the final name is `pagopa-d-weu-wallet-wallet-service-api`

Reply:
I changed the name by removing the **wallet** prefix